requirements.txt

```diff
@@ -0,0 +1,8 @@
+boto3==1.28.0
+idna==1.2.1
+py==1.9.0
+django==2.2.28
```

Comment:
:exclamation:**Cycode**: Security vulnerability found in newly introduced dependency. 

||| 
 |---|---|
|**Severity**|Medium|
|**Issue**|Django denial-of-service attack in the intcomma template filter: CVE-2024-24680|
|**Ecosystem**|PyPI|
|**Dependency**|`django`|
|**Dependency Paths**|`django 2.2.28`|
|**Direct Dependency**|Yes|
|**Upgrade**|`3.2.24`|

 
 An issue was discovered in Django 3.2 before 3.2.24, 4.2 before 4.2.10, and Django 5.0 before 5.0.2. The intcomma template filter was subject to a potential denial-of-service attack when used with very long strings.
 <!-- 14930e4f-b4ca-450b-8e61-49aea9ee9987 -->